source/agora/node/Registry.d

```diff
@@ -1366,51 +1375,100 @@ private struct ZoneData
            type = the type of the record that was requested, default is `ALL`
 
         Returns:
-            TypedPayload with type `type` of name registry associated with
-            `public_key`. All records are returned when `type` is `ALL`
+            ResourceRecord array with type `type` of name registry associated with
```

Comment:
Technically, it will match `type`, but it *is* not type, because `type` can be a wildcard.

Reply:
I only know `ALL` condition and its commented